DiamondLightSource/mx-bluesky
# Update to new exp-eye endpoints

Fixes #636 

To test:
* Confirm unit tests pass
* Confirm system tests pass (`pytest tests/system_tests/hyperion/external_interaction/test_exp_eye_dev.py`)
